resources/cdk/cognito_scenario_user_pool_with_mfa/README.md

```diff
@@ -0,0 +1,118 @@
+# Cognito user pool with MFA scenario resources
+
+## Overview
+
+Creates AWS resources for Amazon Cognito scenarios. The scripts in this
+example create the following resources:
+ 
+* An Amazon Cognito user pool that is configured to:
+    * allow self sign-up
+    * verify by email
+    * require multi-factor authentication (MFA)
+    * allow device tracking
```

Comment:
* An Amazon Cognito user pool that is configured to do the following:
 *   * Allow self sign-up
 *   * Verify by email
 *   * Require multi-factor authentication (MFA)
 *   * Allow device tracking

Reply:
fixed